doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,502 @@
+* When running the :ref:`bluetooth_central_dfu_smp` sample, the :option:`CONFIG_BT_SMP` configuration must be aligned between this sample and the Zephyr counterpart (:ref:`zephyr:smp_svr_sample`).
+  However, security is not enabled by default in the Zephyr sample.
+* The central samples (:ref:`central_uart`, :ref:`bluetooth_central_hids`) do not support any pairing methods with MITM protection.
+* On some operating systems, the nrf_desktop application is unable to reconnect to a host.
+* central_uart: A too long 212-byte string cannot be handled when entered to the console to send to peripheral_uart.
+* On nRF51 devices, BLE samples that use GPIO might crash when buttons are pressed frequently.
+  In such case, the GPIO ISR introduces latency that violates real-time requirements of the Radio ISR.
+  nRF51 is more sensitive to this issue than nRF52 (faster core).
+
+
+Bootloader
+----------
+
+* Building and programming the immutable bootloader (see :ref:`ug_bootloader`) is not supported in SEGGER Embedded Studio.
+* The immutable bootloader can only be used with the following boards:
+
+  * nrf52840_pca10056
+  * nrf9160_pca10090
+
+  It does not work properly on nRF51 and nRF53.
+
+
+DFU and FOTA
+------------
+
+* When using :ref:`lib_aws_fota`, no new jobs are received on the device if the device is reset during a firmware upgrade or loses the MQTT connection.
+  As a workaround, delete the stalled in progress job from AWS IoT.
+* :ref:`lib_fota_download` does not resume a download if the device loses the connection.
+  As a workaround, call :cpp:func:`fota_download_start` again with the same arguments when the connection is re-established to resume the download.
+* When using the mcuboot target in :ref:`lib_dfu_target`, the write/downloaded offset is not retained when the device is reset.
+* In the :ref:`aws_fota_sample` and :ref:`http_application_update_sample` samples, the download is stopped if the socket connection times out before the modem can delete the modem firmware.
+  As a workaround, call :cpp:func:`fota_download_start` again with the same arguments.
+  A fix for this issue is available in commit `38625ba7 <https://github.com/NordicPlayground/fw-nrfconnect-nrf/commit/38625ba775adda3cdc7dbf516eeb3943c7403227>`_.
+* If the last fragment of a :ref:`lib_fota_download` is received but is corrupted, or if the last write is unsuccessful, the library emits an error event as expected.
+  However, it also emits an apply/request update event, even though the downloaded data is invalid.
+
+NFC
+---
+
+* NFC tag samples are unstable when exhaustively tested (performing many repeated read and/or write operations).
+  NFC tag data might be corrupted.
+
+Build system
+============
+
+* It is not possible to build and program :ref:`secure_partition_manager` and the application individually.
+
+nrfxlib
+=======
+
+* In the BSD library, the GNSS sockets implementation is experimental.
+
+
+nrfx v1.8.1
```

Comment:
nrfx is in version  v2.1.0. @kl-cruz  can you confirm that these known issues are no longer valid?

Reply:
I can confirm these issues have been fixed in nrfx v2.1.0

Reply:
Speaking of known issues in nrfx. Could you please mention incorrect pin definition in MDK? The number of P1 port pins is wrong for nRF5340. I created a known issue for NFC that manifests itself because of it:

https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1854/files#r378233002

I think it is worth mentioning somewhere. What's your take on it?

Reply:
@kapi-no I think it's worth mentioning here. It would go like this:
* nRF5340:
  Pins P1.12 to P1.16 are unavailable due to incorrect pin number definition in MDK

Reply:
Sorry, in fact it is P1.12 to P1.15, so here is corrected version:
* nRF5340:
  Pins P1.12 to P1.15 are unavailable due to incorrect pin number definition in MDK

Reply:
Removed nrfx section and added an "MDK (part of nrfx)" section.